lib/components/chat_panel.dart

```diff
@@ -106,6 +107,21 @@ DateTime? _getExpiration(
       return null;
     }
     return model.endTimestamp.add(pollEventConfig.eventDuration);
+  } else if (model is TwitchChannelPointRedemptionEventModel) {
+    final channelPointRedemptionEventConfig =
+        eventSubConfigurationModel.channelPointRedemptionEventConfig;
+    final unfulfilledDuration =
+        channelPointRedemptionEventConfig.eventDuration +
+            channelPointRedemptionEventConfig.unfulfilledAdditionalDuration;
+    if (model.status == 'unfulfilled' && unfulfilledDuration > Duration.zero) {
```

Comment:
Can we parse `status` into an enum in the model so we don't have to do string comparisons?